mobile_back_pixel/cpp/backend_tflite/tflite_settings_pixel.h

```diff
@@ -29,55 +29,55 @@ common_setting {
 
 benchmark_setting {
   benchmark_id: "image_classification"
-  accelerator: "nnapi"
-  accelerator_desc: "NNAPI"
-  configuration: "TFLite"
+  accelerator: "tpu"
```

Comment:
`accelerator` is not a human-readable string that can be changed to match `accelerator_desc`.
It's a string that is used during backend configuration, so changing it changes behavior of backend.

Particularly this change from `nnapi` to `tpu` makes the backend fall back to CPU because `tpu` is not a valid accelerator tag.

See here:
https://github.com/mlcommons/mobile_app_open/blob/375bac4c0400d9312b059ed0d541c45a658bce54/mobile_back_tflite/cpp/backend_tflite/tflite_c.cc#L286

Reply:
Same goes for all the other changes of `accelerator` field in this PR.

Reply:
You could fix it by editing the backend sources.
But I think it would be better to revert the change.

`nnapi` value reflects the fact that backend uses `tflite::StatefulNnApiDelegate::Options()` during configuration.

`tpu` in pixel config or `gpu+dsp+npu` in tflite android config is just a guess about possible implementation, which may be wrong.
We can probably still use such guesses for `accelerator_desc` field, but I don't think it's suitable for the `accelerator` field.

Reply:
Using `npu` seems OK to me.

Reply:
Updated